src/nnue/nnue_architecture.h

```diff
@@ -104,12 +104,12 @@ struct Network
       }
     };
 
-#if defined(ALIGNAS_ON_STACK_VARIABLES_BROKEN)
-    static thread_local char bufferRaw[sizeof(Buffer) + alignment];
-    static thread_local char* bufferRawAligned = align_ptr_up<alignment>(&bufferRaw[0]);
-    static thread_local Buffer& buffer = *(new (bufferRawAligned) Buffer);
+#if defined(ALIGNAS_ON_STACK_VARIABLES_BROKEN) || (defined(__clang__) && (__APPLE__))
```

Comment:
I don't think we need ALIGNAS_ON_STACK_VARIABLES_BROKEN because ```alignas(CacheLineSize) static thread_local Buffer buffer;``` is not on the stack.  I tested this w/ gcc 8.3.0